app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
```

Comment:
- [ ] OPTIONAL
It is a nice idea to use one of the Rails Active Callbacks `after_save` to call the `update_post_likes_counter method every time a new like has been saved to database, then make the `update_post_likes_counter` method private, similar to:

```ruby
 # app/models/likes.rb
  after_save :update_like_counter

  private

  def update_like_counter
    author.increment!(:likes_counter)
  end
```